result/go/osv_scanner.txt

```diff
@@ -0,0 +1,30 @@
+Scanning dir ./docker/go/
+Scanned /usr/local/google/home/gongh/Desktop/code/DockerTester/docker/go/base/go.mod file and found 243 packages
+Scanned /usr/local/google/home/gongh/Desktop/code/DockerTester/docker/go/final/go.mod file and found 29 packages
++-------------------------------------+------+-----------+----------------------------------+----------+------------------------+
+| OSV URL                             | CVSS | ECOSYSTEM | PACKAGE                          | VERSION  | SOURCE                 |
++-------------------------------------+------+-----------+----------------------------------+----------+------------------------+
+| https://osv.dev/GHSA-v84f-6r39-cpfc | 6.8  | Go        | github.com/hashicorp/vault       | 1.14.1   | docker/go/base/go.mod  |
+| https://osv.dev/GHSA-4qhc-v8r6-8vwm | 7.5  | Go        | github.com/hashicorp/vault       | 1.14.1   | docker/go/base/go.mod  |
+| https://osv.dev/GHSA-6p62-6cg9-f5f5 | 7.5  | Go        | github.com/hashicorp/vault       | 1.14.1   | docker/go/base/go.mod  |
+| https://osv.dev/GO-2023-2399        |      |           |                                  |          |                        |
+| https://osv.dev/GHSA-mhpq-9638-x6pw | 5.3  | Go        | github.com/dvsekhvalnov/jose2go  | 1.5.0    | docker/go/final/go.mod |
+| https://osv.dev/GO-2023-2409        |      |           |                                  |          |                        |
+| https://osv.dev/GHSA-h3q2-8whx-c29h | 5.5  | Go        | github.com/goreleaser/goreleaser | 1.23.0   | docker/go/final/go.mod |
++-------------------------------------+------+-----------+----------------------------------+----------+------------------------+
+| Uncalled vulnerabilities            |      |           |                                  |          |                        |
++-------------------------------------+------+-----------+----------------------------------+----------+------------------------+
+| https://osv.dev/GO-2022-0646        |      | Go        | github.com/aws/aws-sdk-go        | 1.44.268 | docker/go/base/go.mod  |
```

Comment:
I feel this is a false alarm. Although [GO-2022-0646](https://osv.dev/vulnerability/GO-2022-0646) doesn't specify the fixed version, all its aliases show the fixed version was `1.34.0`. Other tools also didn't detect this vuln. @another-rex How do you think?